Makefile

```diff
@@ -1,22 +1,35 @@
 export GO15VENDOREXPERIMENT := 1
 
+
 all:
-	hack/dockerized "./hack/check.sh && KUBEVIRT_VERSION=${KUBEVIRT_VERSION} ./hack/build-go.sh install ${WHAT} && ./hack/build-copy-artifacts.sh ${WHAT} && DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} IMAGE_PULL_POLICY=${IMAGE_PULL_POLICY} VERBOSITY=${VERBOSITY} ./hack/build-manifests.sh"
+	hack/dockerized "DOCKER_PREFIX=${DOCKER_PREFIX} DOCKER_TAG=${DOCKER_TAG} IMAGE_PULL_POLICY=${IMAGE_PULL_POLICY} VERBOSITY=${VERBOSITY} ./hack/build-manifests.sh && \
+	    hack/bazel-fmt.sh && bazel build \
+		--platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
```

Comment:
We can move this boilerplate:

```
--platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 \
--workspace_status_command=./hack/print-workspace-status.sh \
```

into our .bazelrc file. Will create another PR after this one.

Reply:
Yes it can be great but will create a problem if we will want to support additional platforms.

Reply:
Yep we would have to adapt it. However we only support amd64 right now.